docs/DeveloperGuide.md

```diff
@@ -1,25 +1,91 @@
 # Developer Guide
 
+ ## Introduction 
+ ### What is Diliveri
+Diliveri is desktop application built on Java and has cross platform abilities across different operating systems. 
+With the rise of demand for deliveryman due to the pandemic, this app optimises the delivery route for each delivery 
+man taking into account the number of deliveries per location and distances between locations, we improve the time and 
+efficiency of deliveries. This app aims to be a one stop solution and more for to meet all the needs of a deliverymen.  
```

Comment:
I like how it is very clear who the target audience of the app is, what it does and why it's important. Great introduction